docs/content/docs/dev/table/sourcesSinks.md

```diff
@@ -249,9 +249,9 @@ When writing a dynamic table, the content can always be considered as a changelo
 for which all changes are written out continuously until the changelog is exhausted. The returned _changelog mode_
 indicates the set of changes that the sink accepts during runtime.
 
-For regular batch scenarios, the sink can solely accept insert-only rows and write out bounded streams.
+For regular batch scenarios, the sink can only accept insert-only rows and write out bounded streams.
 
-For regular streaming scenarios, the sink can solely accept insert-only rows and can write out unbounded streams.
```

Comment:
Thanks @mans2singh for the contribution, I think current description is fine to me, is there any special reason that we have to avoid use the word `solely`?

Reply:
Hi @leonardBang - I have reverted that change.  Please let me know if the second part is ok.  Thanks for your time.